spec/unit/util/execution_spec.rb

```diff
@@ -119,6 +119,17 @@ def stub_process_wait(exitstatus)
         call_exec_posix("/bin/echo 'foo' ; \n /bin/echo 'bar' ;", {:uid => 50, :gid => 55}, @stdin, @stdout, @stderr)
       end
 
+      context 'cwd option' do
+        let(:cwd) { 'cwd' }
+
+        it 'should run the command in the specified working directory' do
```

Comment:
There are similar unit tests to the AIX PR here @ekinanp: these are using `expects` and specifying specific parameters are going to be used, which will test implementation rather than function.

Reply:
Discussed this on HipChat -- `Dir.expects(:chdir).with(cwd)` is fine b/c that is required to enforce the functionality, while `Kernel.expects(:exec).with('test command')` can be changed to `stubs`, but we will keep it here to make it consistent with the other unit tests.